.github/workflows/stale-actions.yml

```
@@ -3,9 +3,13 @@ on:
   schedule:
   - cron: "00 10 * * *"
 
+permissions: read-all
+
 jobs:
   stale:
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
```

```suggestion
      issues: write
      pull-requests: write
```

Isn't it needed also `pull-requests`?

---

Thanks, I've overlooked it.